templates/aws/index.html

```diff
@@ -30,7 +30,39 @@ <h1>Ubuntu on AWS</h1>
     </div>
   </div>
 </section>
-
+<section class="p-strip--square-suru">
+  <div class="row">
+    <div class="col-9">
+      <h2 class="p-heading--3">Meet our Public Sector team at AWS Public Secctor Summit</h2>
+      <p>May 23 &ndash; 24 at Walter E. Waashington Convention Center, Washington DC</p>
+    </div>
+    <div class="col-3">
+      <a href="/engage/aws-summit-washington-DC">
```

Comment:
It's not obvious that these are links, we should include an explicit CTA i.e. `a.p-button--positive` instead - it's not in the copy doc, but I'd propose:

```html
<a href="/engage/aws-summit-washington-DC" class="p-button--positive">Meet the team</a>
```